source/lmp/tests/test_lammps.py

```
@@ -719,3 +721,32 @@ def test_pair_deepmd_mpi(balance_args: list) -> None:
+
+@pytest.mark.skipif(
+    shutil.which("mpirun") is None, reason="MPI is not installed on this system"
+)
+@pytest.mark.skipif(
+    importlib.util.find_spec("mpi4py") is None, reason="mpi4py is not installed"
+)
+def test_pair_deepmd_mpi_subcommunicators(tmp_path: Path) -> None:
+    """Independent LAMMPS communicators must load their requested models."""
+    output_a = tmp_path / "forces_a.out"
+    output_b = tmp_path / "forces_b.out"
+    sp.check_call(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)